src/components/types/Tree.ts

```diff
@@ -0,0 +1,10 @@
+export type TreeNode = {
+  label: string
```

Comment:
Should we also have a `nodeKey` in tree (like `rowKey` in list) since titles of 2 nodes could be same